code/control/control.cpp

```diff
@@ -0,0 +1,21 @@
+#include <cstdio>
+
+int isodd(int i) { return i % 2; }
```

Comment:
```suggestion
bool isodd(int i) { return i % 2; }
```

Reply:
As you found out below the reason why it is int is that the move to the switch statement won't work. But, as we privately discussed, maybe a good solution is to have it bool here and in the description when asking the students to move to the switch statement tell them that the "isodd" function won't work anymore and they would need to slightly change it.